detection_rules/config.py

```diff
@@ -284,6 +285,14 @@ def parse_rules_config(path: Optional[Path] = None) -> RulesConfig:
     # kql keyword normalization
     contents['normalize_kql_keywords'] = loaded.get('normalize_kql_keywords', True)
 
+    if loaded.get('auto_gen_schema_file'):
+        contents['auto_gen_schema_file'] = base_dir.joinpath(loaded['auto_gen_schema_file'])
+
+        # Check if the file exists
+        if not contents['auto_gen_schema_file'].exists():
```

Comment:
You should update the docs and example files to include this new field/feature.

Reply:
Updated, will update [here](https://github.com/elastic/DaC-Reference/pull/16) as well.